xml/System.Runtime.Intrinsics/Vector128.xml

```diff
@@ -1766,9 +1768,9 @@
         <Parameter Name="vector" Type="System.Runtime.Intrinsics.Vector128&lt;System.Single&gt;" Index="0" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="vector">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="vector">The vector to convert.</param>
+        <summary>Converts a <see cref="T:System.Runtime.Intrinsics.Vector128`1" /> to a <see cref="T:System.Runtime.Intrinsics.Vector128`1" /> platform specific behavior on overflow.</summary>
```

Comment:
```suggestion
        <summary>Converts a <see langword="Vector128{Single}" /> to a <see langword="Vector128{Int32}" /> using platform-specific behavior on overflow.</summary>
```

Reply:
The type parameters from https://github.com/dotnet/runtime/blob/6af853d7757e55a09fd9bc6dc1d6aa12c0b454f2/src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs#L572 were lost here - and similarly throughout these APIs.

Reply:
I'll have to fix this later. Need to fix the bug in the tool first. Is that ok?